framework/include/base/Coupleable.h

```diff
@@ -344,6 +351,14 @@ class Coupleable
   /// This will always be zero because the default values for optionally coupled variables is always constant
   VariableSecond _default_second;
 
+  /**
+   * Check that the right kind of variable is being coupled in
+   *
+   * @param var_name The name of the coupled variable
+   * @param comp The component of the coupled variable
+   */
+  void checkVar(const std::string & var_name, unsigned int comp);
```

Comment:
I managed to sneak in some [beer code](http://www.czechvar.com/) in here

Reply:
Haha! Nice you should have been a little more blatant and actually spelled it that way too!